### PR TITLE
Automated Install Scripts 

### PR DESCRIPTION
**Pull request by raymondberg**
Hey @dewyze, I made some more install automation based on your dotfiles.
- Automated install master using just `rake` or `rake install` or `rake all:install`
- Installer ensures folders are set for vundler and auto pulls it. You could possible prompt for vimplug later.
- Added bash_aliases and .bashrc files
- Snarky Readme

### CODE REVIEW

**Comment by dewyze** [comment]
I'm going to look at this soon, I swear!

File: aliases_shared [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+alias cd3.="cd ../../.."
+alias cd4.="cd ../../../.."
+alias cd5.="cd ../../../../.."
+alias cd6.="cd ../../../../../.."
+
+alias ll='ls -alF'
+alias la='ls -A'
+alias l='ls -CF'
+
+bmux_start () {
```
Comment (dewyze) [inline aliases_shared:12]: @raymondberg why bmux? Honorable nod to its origin?

Comment (raymondberg) [inline aliases_shared:12]: It's true; also a short version without overwriting tmux.

On Monday, October 26, 2015, John DeWyze notifications@github.com wrote:

> In aliases_shared
> https://github.com/dewyze/dotfiles/pull/1#discussion_r43072881:
> 
> > @@ -0,0 +1,29 @@
> > +alias cd.="cd .."
> > +alias cd2.="cd ../.."
> > +alias cd3.="cd ../../.."
> > +alias cd4.="cd ../../../.."
> > +alias cd5.="cd ../../../../.."
> > +alias cd6.="cd ../../../../../.."
> > +
> > +alias ll='ls -alF'
> > +alias la='ls -A'
> > +alias l='ls -CF'
> > +
> > +bmux_start () {
> 
> @raymondberg https://github.com/raymondberg why bmux? Honorable nod to
> its origin?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dewyze/dotfiles/pull/1/files#r43072881.

## 

Respectfully,
Ray Berg

File: bashrc [hunk 2]
```diff
@@ -0,0 +1,123 @@
+# ~/.bashrc: executed by bash(1) for non-login shells.
```
Comment (dewyze) [inline bashrc:1]: I don't use bash, I am going to trust these all work.

**Comment by dewyze** [comment]
If you could rebase off of master I will merge this in.

File: bashrc (hunk 2 above)
Comment (raymondberg) [inline bashrc:1]: Why worry?

**Comment by raymondberg** [comment]
@dewyze I've rebased on your branch. Do it! :)

**Comment by dewyze** [comment]
@raymondberg donezo!

**Pull request merged**